nameable.rb

```diff
@@ -0,0 +1,7 @@
+# Implement a method called correct_name that will raise a NotImplementedError.
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages capitalized to make your pull request professional 👍

Reply:
@HAMMAS-SALEEM, I make it on the future commits, thanks for your feedback.